language/use-cases/document-qa/question_answering_documents_langchain_bigquery_vector_search.ipynb

```diff
@@ -0,0 +1,1026 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    results = retrieval_qa.invoke({"query": search_query})
</pre><p>Why are you adding the <code>search_query</code> into a dictionary in this call, but you didn't earlier? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
Changed it to make it consistent with the above.